spec/rabbit_active_record_callbacks_spec.rb

```diff
@@ -28,5 +29,13 @@ class Foo < ActiveRecord::Base
         f.update_attributes(word: :bar)
       end
     end
+
+    context 'when deleting a record' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.